src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_config_importer.cljs

```diff
@@ -149,24 +154,34 @@
    (fn [{:keys [state props]}]
      [:div {}
       (cond
-        (:show-import-mc-modal? @state) [ConfigurationImportForm {:selected-method-config (:selected-method-config @state)
-                                                                  :workspace-id (:workspace-id props)
-                                                                  :on-import (fn [& args]
-                                                                               (swap! state dissoc :show-import-mc-modal?)
-                                                                               (apply (:on-import props) args))}]
-        (:loaded-import-confs? @state) [ConfigurationsTable {:method-configs (:method-configs @state)
+        (:show-import-mc-modal? @state) [ConfigurationImportForm
+                                         {:selected-method-config (:selected-method-config @state)
+                                          :workspace-id (:workspace-id props)
+                                          :on-back (fn []
+                                                     (swap! state dissoc
+                                                       :show-import-mc-modal?
+                                                       :selected-method-config)
+                                                     (swap! state assoc
+                                                       :loaded-import-confs? true))
+                                          :on-import (fn [& args]
```

Comment:
I don't know why either.  But I am left everything alone here except for whitespace.  Would such a refactoring be outside the scope of this PR?